packages/legend-extension-external-store-service/package.json

```diff
@@ -48,6 +48,7 @@
     "mobx-react-lite": "3.2.2",
     "monaco-editor": "0.30.1",
     "react": "17.0.2",
+    "react-icons": "4.3.1",
```

Comment:
Let's not try to import directly from `react-icons` but use `Icon.tsx` in `@finos/legend-art`